client/components/domains/connect-domain-step/index.jsx

```diff
@@ -61,11 +61,16 @@ function ConnectDomainStep( {
 	const [ loadingDomainSetupInfo, setLoadingDomainSetupInfo ] = useState( false );
 
 	const baseClassName = 'connect-domain-step';
-	const isStepStart = stepType.START === stepsDefinition[ pageSlug ].step;
-	const mode = stepsDefinition[ pageSlug ].mode;
-	const step = stepsDefinition[ pageSlug ].step;
-	const prevPageSlug = stepsDefinition[ pageSlug ].prev;
-	const isTwoColumnLayout = ! stepsDefinition[ pageSlug ].singleColumnLayout;
+	if ( stepsDefinition[ pageSlug ] === undefined ) {
+		// eslint-disable-next-line no-console
+		console.error( 'Tried to set invalid pageSlug in ConnectDomainStep', pageSlug, firstStep );
+	}
+	const currentStep = stepsDefinition[ pageSlug ] || stepsDefinition[ firstStep ];
```

Comment:
I wonder if a user can get stuck in the first step due to some error with `pageSlug` 🤔 but I guess it's better than a WSOD